# This comment has "quotes" in it and \slashes\
rosgraph_msgs/Log[4] a